<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectDialog</class>
 <widget class="QDialog" name="projectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Description</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Job No.              :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Structure Tag / Item :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Designed by        :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="job_no_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="item_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="engineer_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Title        :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Symbol   :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="project_symbol_edit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="project_title_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Client / Owner    :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="client_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rev. No. :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rev_edit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>job_no_edit</tabstop>
  <tabstop>project_symbol_edit</tabstop>
  <tabstop>project_title_edit</tabstop>
  <tabstop>client_edit</tabstop>
  <tabstop>item_edit</tabstop>
  <tabstop>engineer_edit</tabstop>
  <tabstop>rev_edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>projectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>projectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
